Controllers/ItemsController.cs

```diff
@@ -44,31 +68,28 @@ public async Task<IActionResult> Details(int? id)
             return View(item);
         }
 
-        // GET: Items/Create
         [Authorize]
         public IActionResult Create()
         {
             return View();
         }
 
-        // POST: Items/Create
-        // To protect from overposting attacks, enable the specific properties you want to bind to.
-        // For more details, see http://go.microsoft.com/fwlink/?LinkId=317598.
         [Authorize]
         [HttpPost]
         [ValidateAntiForgeryToken]
-        public async Task<IActionResult> Create([Bind("ID,DateOfPurchase,Condition,Category,UserID,Length,Height,Width,Name,Description,KarmaPoints")] Item item)
+        public async Task<IActionResult> Create([Bind("ID,DateOfPurchase,Condition,Category,Length,Height,Width,Name,Description,KarmaPoints")] Item item)
```

Comment:
Hmm, where da userid gone? Or i have missed something somewhere and we dont need it anymore?

Reply:
Nevermind, already saw an answer :)

Reply:
ye its set automatically